107/src/main/java/org/ehcache/jsr107/DefaultJsr107SerializationProvider.java

```diff
@@ -24,15 +25,6 @@
 public class DefaultJsr107SerializationProvider extends DefaultSerializationProvider {
 
   public DefaultJsr107SerializationProvider() {
-    super(null);
+    super(new DefaultSerializationProviderConfiguration().addSerializerFor(Object.class, (Class) JavaSerializer.class));
```

Comment:
Can't we use the new compact one? And ditch that `JavaSerializer` from the code base altogether.